src/controllers/index.controller.js

```diff
@@ -155,6 +201,7 @@ async function start(req, res) {
         multisigData,
         passphrase: req.body.passphrase,
         allowPassphrase: config.allowPassphrase,
+        scanPolicy: scanPolicyData,
```

Comment:
[⭕ verify] The `scanPolicyData` should be validated inside the `getReadlongWalletConfig` because it can be evaluated to non integer value after `parseInt`.